core/sql/regress/charsets/TEST315

```diff
@@ -315,6 +315,26 @@ select converttohex(_ucs2 x'515a515b515c515d') from dual;
 select char_length(reverse(_ucs2 x'515a515b515c515d')) from dual;
 select converttohex(reverse(_ucs2 x'515a515b515c515d')) from dual;
 
+-- tests for OVERLAY function
+select overlay('w1234567xyz' placing 'abcde' from 3)  from dual;
+select overlay('w1234567xyz' placing 'abcde' from 3 for 4)  from dual;
```

Comment:
Do we need a test case to cover the example in the description  where numeric length is 0?

> select overlay('w1234567xyz' placing 'abcde' from 3 for 0)  from dual;